api/circle/landing/[token].ts

```diff
@@ -0,0 +1,62 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { adminClient } from '../../../api-lib/gql/adminClient';
+import { errorResponse } from '../../../api-lib/HttpError';
+import { Awaited } from '../../../api-lib/ts4.5shim';
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  // TODO: does this have some kind of CORS protection?
```

Comment:
say more about this - I approached a CORS attack with SIWE where I just checked the origin URL.

Reply:
I think this is a non-issue, the API routes are CORS protected by default i believe